apps/andi/config/integration.exs

```diff
@@ -12,11 +12,20 @@ endpoint = [{host, 9092}]
 config :andi,
   divo: [
     {DivoKafka, [create_topics: "event-stream:1:1", outside_host: host, kafka_image_version: "2.12-2.1.1"]},
-    {DivoRedis, []}
+    {DivoRedis, []},
+    {Andi.DivoPostgres, []}
   ],
   divo_wait: [dwell: 700, max_tries: 50],
   kafka_broker: endpoint
 
+config :andi, Andi.Repo,
+  database: "andi",
+  username: "postgres",
+  password: "postgres",
```

Comment:
Probably want to change the two values above and make them environment variables.

Reply:
These are just config values so we can spin up ecto in the integration environment. This is how we are doing it in discovery_api

Reply:
this is just used for integration tests. No need to worry about security